testsuite/libra-swarm/src/main.rs

```diff
@@ -74,7 +87,7 @@ fn main() {
     println!("To run the Libra CLI client in a separate process and connect to the validator nodes you just spawned, use this command:");
 
     println!(
-        "\tcargo run --bin cli -- -u {} -m {:?} --waypoint {} --chain-id {:?}",
+        "\tcli -u {} -m {:?} --waypoint {} --chain-id {:?}",
```

Comment:
Could you elaborate on this and the other changes that remove "cargo run". I'm guessing because you imagine that these would be installed?

Reply:
The idea should be that swarm is a binary/library that can be used to orchestrate local instances of a network. Theres been a ton of interest in the past for allowing for this to be run outside of the repo and because of that you can't tell people to run cargo (this does bring up the point that our binaries have terrible names 😄 )

Reply:
I'm a little curious about this one... I wish we had metrics on what people are using and building :). I've been able to reason about most of my concerns. Could we update search for and update the readmes for swarm? I think there's some on the website.

Reply:
Yeah...thinking about this a little more maybe there's a bit more that should be done before this lands. I'm thinking maybe at the very least providing a wrapper around `libra-swarm` that has the exact same behavior we have today. Let me see what I can do in the next couple of days and I'll have you take 1 more look at it then